include/sacn/receiver.h

```diff
@@ -111,17 +129,43 @@ typedef struct SacnLostSource
  * @{
  */
 /*! Filter preview data. If set, any sACN data with the Preview flag set will be dropped for this
- *  universe and sources sending only Preview data will not be tracked. */
+ *  universe but sources sending only Preview data will still trigger a SacnSourcesFoundCallback(). */
 #define SACN_RECEIVER_OPTS_FILTER_PREVIEW_DATA 0x1
 /*!
  * @}
  */
 
+/*!
+ * \brief Notify that one or more sources have been found.
+ *
+ * New sources have been found that can fit in the current collection.  The DMX data and per-address priorities for each
+ * source may be acted upon immediately, as the library has determined the correct starting values.  Additionally, the
+ * library has waited for a "sampling period" upon startup to make sure the starting set of sources is consistent.
+ *
+ * After this callback returns, packets for this source will be sent to the sACNUniverseDataCallback().
+ *
+ * \param[in] handle Handle to the receiver instance for which sources were found.
+ * \param[in] found_sources Array of structs describing the source or sources that have been found with their current
+ * values.
+ * \param[in] num_sources_found Size of the found_sources array.
+ * \param[in] context Context pointer that was given at the creation of the receiver instance.
+ */
+typedef void (*SacnSourcesFoundCallback)(sacn_receiver_t handle, const SacnFoundSource* found_sources,
+                                         size_t num_found_sources, void* context);
+
 /*!
  * \brief Notify that a data packet has been received.
  *
- * Will be called for every sACN data packet received on a listening universe, unless the
- * Stream_Terminated bit is set.
+ * Will be called for every sACN data packet received on a listening universe for a found source, unless the
+ * Stream_Terminated bit is set or if preview packets are being filtered.
+ *
+ * The callback will only be called for packets whose sources have been found via SacnSourcesFoundCallback(), and have
+ * not been lost via SacnSourcesLostCallback().  It will be called for all data packets received, even those without
+ * a startcode of 0 or 0xdd.
+ *
+ * If the source is sending sACN Sync packets, this callback will only be called when the sync packet is received,
+ * if the source forces the packet, or if the source sends a data packet without a sync universe.
+ * TODO: We still need add support for sACN Sync.
```

Comment:
This is public documentation, I would avoid using "we" here and just say something like:

> NOTE: this version of the sACN library does not support sACN Sync. This paragraph will be valid in the future.